hw/ip/otbn/dv/model/iss_wrapper.cc

```diff
@@ -99,15 +99,18 @@ struct TmpDir {
   }
 };
 
-// Find the otbn Python model, based on our executable path, and
-// return it. On failure, throw a std::runtime_error with a
-// description of what went wrong.
+// Find the top of the OpenTitan repository
 //
-// This works by searching upwards from the binary location to find a git
-// directory (which is assumed to be the OpenTitan toplevel). It won't work if
-// you copy the binary somewhere else: if we need to support that sort of
-// thing, we'll have to figure out a "proper" installation procedure.
-static std::string find_otbn_model() {
+// If REPO_TOP is defined, use that. Otherwise, this will only work if we're
+// running from a binary inside the git repository. This happens with the
+// default paths (which put BUILD_BIN at $REPO_TOP/build-bin). If we can't find
+// an enclosing repository, throw a std::runtime_error.
+static std::string find_repo_top() {
+  const char *from_env = getenv("REPO_TOP");
```

Comment:
Note that OTBN regressions in our nightly will fail, because the compute machines that run the simulation does not have access to our repo. We need to fix DVsim to copy over the entire repo to the scratch area. Currently, we only copy over the necessary tool sources.

Reply:
Yep, but that's something a Google engineer will have to do. I didn't see much point in coordinating for this - we'll get "0% pass" lines for this IP block for a couple of days, but that doesn't really matter (the lines weren't there at all before!)